src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --main-color: #000;
+  --link-color: #00acdc;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px ;
+}
+
+.logo {
+  display: block;
+  line-height: 0;
+}
+
+.logo__image {
+  display: block;
+  width: 100%;
+  object-fit: cover;
+  height: auto;
+}
+
+.nav {
+  font-size: 12px;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  text-transform: uppercase;
   margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  position: relative;
+  margin-left: 20px;
+
+}
+
+.nav__link {
+  display: block;
+  line-height: 60px;
+  text-decoration: none;
+  color: var(--main-color);
```

Comment:
This is good! You are reusing the color from variables which is a way to do in a real project. 👍